packages/manager/modules/vps/src/dashboard/tile/configuration/index.html

```diff
@@ -30,6 +31,16 @@
                     ></span>
                 </a>
             </p>
+            <oui-message
+                type="info"
+                data-ng-if="$ctrl.serviceInfos.contactAdmin !== $ctrl.nichandle"
+            >
+                <p
+                    class="mb-0"
+                    data-translate="vps_dashboard_tile_configuration_upgrade_wrong_nic_cpu"
+                    data-translate-values="{ nichandle: '<strong>' + $ctrl.serviceInfos.contactAdmin + '</strong>' }"
+                ></p>
+            </oui-message>
```

Comment:
May be create a component could be usefull ?

Reply:
Maybe in second time, the first priority is unblock the tech customer